index.html

```diff
@@ -67,11 +66,167 @@ <h3>
+          </div>
+        </div>
+        <div class="work-5">
+          <div class="img-holder"></div>
+          <div class="posts">
+            <h2>Multi-Post Stories Gain+Glory</h2>
+            <div class="flex-post">
+              <ul>
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">HTML</a></li>
+              </ul>
+            </div>
+            <button type="button" class="works-button">See Project</button>
+          </div>
+        </div>
+        <div class="work-6">
+          <div class="img-holder"></div>
+          <div class="posts">
+            <h2>Multi-Post Stories Gain+Glory</h2>
+            <div class="flex-post">
+              <ul>
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">HTML</a></li>
+              </ul>
+            </div>
+            <button type="button" class="works-button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <section class="about-me">
+      <div class="about-me-text">
+        <h2>About Me</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button type="button" class="get-resume">Get My Resume</button>
+      </div>
+    </section>
+    <div class="my-knowledge">
+      <section class="languages">
+        <img src="images/lang.svg" alt="languages" />
+        <h2>Languages</h2>
+        <div class="flex-languages">
+          <ul>
+            <li><a href="#">JavaScript</a></li>
+            <li><a href="#">Ruby</a></li>
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">CSS</a></li>
+          </ul>
+        </div>
+      </section>
+      <section class="frameworks">
+        <img src="images/frameworks.svg" alt="frameworks" />
+        <h2>Frameworks</h2>
+        <div class="flex-frameworks">
+          <ul>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby on Rails</a></li>
+            <li><a href="#">RSpec</a></li>
+            <li><a href="#">Copybara</a></li>
+            <li><a href="#">Selenium</a></li>
+          </ul>
+        </div>
+      </section>
+      <section class="skills">
+        <img src="images/skills.svg" alt="skills" />
+        <h2>Skills</h2>
+        <div class="flex-skills">
+          <ul>
+            <li><a href="#">Codekit</a></li>
+            <li><a href="#">Github</a></li>
+            <li><a href="#">Codepen</a></li>
+            <li><a href="#">Gitlab</a></li>
+            <li><a href="#">Terminal</a></li>
+          </ul>
```

Comment:
- [ ] Kindly unwrap the list from a tag because they are just the list of skills that you know. Keep coding 💪